### PR TITLE
Heroes: Add bleeding image support

### PR DESCRIPTION
**Pull request by Scotchester**
This PR enables hero images that bleed off the top and bottom of the hero container when in the two-column layout.

## Additions

- A new `__bleed-vertical` modifier to trigger the various vertical bleed styles.

## Changes

- Major (as in breaking) refactor of heroes, generally. The big changes here relate to updating the markup for bleed to work for a variable hero height and enabling an alternate image for the single-column layout.

## Testing

- See jimmynotjim/capital-framework-sandbox#16

## Review

- @cfpb/front-end-team-admin 

### Max width
screen shot 2016-08-12 at 17 16 31

### Min width at two columns
screen shot 2016-08-12 at 17 17 56

### Single-column layout with alternate image
screen shot 2016-08-12 at 17 18 43

## Todos

- Add more examples to usage.md

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by KimberlyMunoz** [comment]
Hey Scott, I think this PR might need to be rebased before it can be reviewed since `prerelease-ver-4` was rebased to bring in canary changes.

**Comment by Scotchester** [comment]
@KimberlyMunoz Fixed! Sorry for the delay.

File: src/cf-layout/src/molecules/heroes.less [hunk 1]
```diff
@@ -71,53 +76,104 @@
...
-            margin-bottom: unit( @grid_gutter-width / @size-iii, em );
-        } );
+    &_subhead:last-child {
+        margin-bottom: 0;
     }
 
     &_cta {
-        display: block;
-    }
+        // display: block;
```
Comment (KimberlyMunoz) [inline src/cf-layout/src/molecules/heroes.less]: Is this still needed?

File: src/cf-layout/src/molecules/heroes.less [hunk 2]
```diff
@@ -71,53 +76,104 @@
...
 
-    &_image {
-        .grid_column( 1, 1 );
+        &:not( .btn ) {
```
Comment (KimberlyMunoz) [inline src/cf-layout/src/molecules/heroes.less]: Clever. 🐰

**Comment by KimberlyMunoz** [comment]
There's still a whole bunch of my commits at the start of this branch. :'(
image

**Comment by Scotchester** [comment]
If I understand correctly, that happened because I merged in the rebased `prerelease-ver-4` branch, which I did because I started to attempt the rebase and it was hell. Perhaps someone with better Git Fu than me can try to rebase it in a better way than I could?

I can just re-branch and re-apply the changes, too, I suppose.

**Comment by KimberlyMunoz** [comment]
I tried doing this earlier, and honestly, I think opening a new PR and re-branching is way easier.

File: src/cf-layout/src/molecules/heroes.less (hunk 1 above)
Comment (Scotchester) [inline src/cf-layout/src/molecules/heroes.less]: Whoops, no! Will remove in new PR.

**Comment by Scotchester** [comment]
Closing in favor of #387.

**Pull request closed without merging**